### PR TITLE
Pandaria - Main

### PR DESCRIPTION
**Pull request by ghost**
## Warnings:
_**This is the main branch of Pandaria! Don't commit directly to it, create new branches and open PRs instead!**_

## Changelog:
- Added Pandaria continent with its cultures, religions and flavour.

## TODO:
- [x] Add names for Pandarian cultures.
   - [x] Pandaren
   - [x] Mogu
   - [x] Mantid
   - [x] Jinyu
   - [x] Ankoan
   - [x] Saurok
   - [x] Grummle
   - [x] Yaungol
- [x] Add race traits for Pandarian cultures.
   - [x] Pandaren
   - [x] Mogu
   - [x] Mantid
   - [x] Jinyu
   - [x] Ankoan
   - [x] Saurok
   - [x] Grummle
   - [x] Yaungol
- [x] Add portraits for new races
   - [x] Pandaren
   - [x] Mogu
   - [x] Mantid
   - [x] Jinyu
   - [x] Ankoan
   - [x] Saurok
   - [x] Grummle
   - [x] Yaungol
- [x] Add race icons for new races
   - [x] Pandaren
   - [x] Mogu
   - [x] Mantid
   - [x] Jinyu
   - [x] Ankoan
   - [x] Saurok
   - [x] Grummle
   - [x] Yaungol
- [x] Complete history ( #1364 )
- [x] Implement life spans
- [x] Empire tier title of Pandaria culture should be named Empire

## How to test:
TBA

## Notes:
TBA

### CODE REVIEW

File: common/cultures/culture_titanforged.txt [hunk 1]
```diff
@@ -481,6 +481,41 @@ titanforged_group = {
 		mat_grm_name_chance = 40
 		mother_name_chance = 5
 
+		# Modifiers
+		modifier = default_culture_modifier
+	}
+	mogu = {
```
Comment (zumbak04) [inline common/cultures/culture_titanforged.txt:487]: What about the mogu culture being in their own group. Their words and letters barely connected to the Titan language. Pandaren, jinyu, saurok, etc. will be in the mogu group.
https://wow.gamepedia.com/Mogu_(language)#Speculation

File: common/scripted_triggers/wc_scripted_triggers.txt [hunk 2]
```diff
@@ -1013,6 +1019,12 @@ is_medium_humanoid_trigger = {
 			trait = creature_frostnymph
 			trait = creature_highborne
 			trait = creature_sethrak
+			trait = creature_pandaren
+			trait = creature_jinyu
+			trait = creature_ankoan
+			trait = creature_mantid
+			trait = creature_saurok
+			trait = creature_mogu
```
Comment (zumbak04) [inline common/scripted_triggers/wc_scripted_triggers.txt:1027]: Mogu are large humanoids.
https://wow.gamepedia.com/Mogu

File: common/cultures/culture_titanforged.txt (hunk 1 above)
Comment (Malicos) [reply common/cultures/culture_titanforged.txt]: mogu and saurok constitute of one culture now

File: common/religions/elemental_lords_group.txt [hunk 3]
```diff
@@ -111,6 +111,7 @@ elemental_lords_group = {
 			same_religion_opinion = 25
 			rel_head_opinion = 25
 			old_gods_worship_opinion = 25
+			waterspeaker_opinion = 25
```
Comment (zumbak04) [inline common/religions/elemental_lords_group.txt:114]: The opinion not declared.

File: common/religions/pandaren_religion_group.txt [hunk 4]
```diff
@@ -103,6 +103,7 @@ pandaren_religion_group = {
 			global_levy_size = -0.1
 			
 			# Opinions of other religions
+			neptulon_worship_opinion = 25
```
Comment (zumbak04) [inline common/religions/pandaren_religion_group.txt:106]: The opinion not declared.
`neptulon_worship` will have more opinion of `waterspeaker` religion than their own?

File: common/religions/elemental_lords_group.txt (hunk 3 above)
Comment (Malicos) [reply common/religions/elemental_lords_group.txt]: scrapped it entirely

File: common/religions/pandaren_religion_group.txt (hunk 4 above)
Comment (Malicos) [reply common/religions/pandaren_religion_group.txt]: scrapped it entirely

File: common/traits/wc_race_traits.txt [hunk 5]
```diff
@@ -1974,6 +1975,158 @@ creature_ancient_wind = {
...
+	diplomacy = 2
+	combat_rating = 5 	# Martial artists
+	
+	ai_ambition = -5 	# Peaceful
+
+	same_opinion = 10
+	hozen_opinion = 10
+	jinyu_opinion = 10
+	grummle_opinion = 10
+	general_opinion = -10
```
Comment (zumbak04) [inline common/traits/wc_race_traits.txt:1992]: First, it's not localized.
<details>
<summary>Click to expand</summary>

image

</details>
Second, these opinions don't work.
<details>
<summary>Click to expand</summary>

image

</details>
Third, jinyu have +10 opinion in sum of other jinyu.
<details>
<summary>Click to expand</summary>

image

</details>

**Pull request merged**